manila/share/drivers/netapp/dataontap/client/client_cmode.py

```diff
@@ -2369,8 +2369,12 @@ def _get_create_volume_api_args(self, volume_name, thin_provisioned,
         # SAPCC Ignore when logical_space_reporting is None. The attribue
         # will be the same of its parent vserver.
         if logical_space_reporting in (True, False):
-            api_args['is-space-reporting-logical'] = logical_space_reporting
-            api_args['is-space-enforcement-logical'] = logical_space_reporting
+            if logical_space_reporting:
```

Comment:
Would it make sense to apply the same fix for `_create_vserver`  method above?

Reply:
fixed